addons/mail/static/tests/mail_test_helpers_contains.js

```diff
@@ -828,10 +829,10 @@ class Contains {
             return;
         }
         let elems;
-        if (target === getFixture() && document.querySelector(this.selector) === target) {
+        if (target === getFixture() && queryAll(this.selector)[0] === target) {
```

Comment:
quelAll() [0] === queryFirst() ??

Reply:
Oops, I didn't know this function exists. I saw `queryOne()` which has the undesirable effect to throw error immediately when not `exact` count match so that's why I used `queryAll` instead. `queryFirst` =/= `queryOne`.
I'm changing it now